pkgs/development/python-modules/colcon-coveragepy-result/default.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  colcon,
+  coverage,
+  pytest-cov,
+  pytestCheckHook,
+  setuptools,
+  scspell,
+  writableTmpDirAsHomeHook,
+}:
+
+buildPythonPackage rec {
+  pname = "colcon-coveragepy-result";
+  version = "0.0.8";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "colcon";
+    repo = "colcon-coveragepy-result";
+    tag = version;
+    hash = "sha256-+xjrmiWaDPjoRwjgP4Ui6+vuG4Nc4ur8DdC8ddiXAG0=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    colcon
+    coverage
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov
+    pytestCheckHook
+    scspell
+    writableTmpDirAsHomeHook
+  ];
+
+  pythonImportsCheck = [
+    "colcon_coveragepy_result"
+  ];
+
+  disabledTestPaths = [
+    "test/test_flake8.py"
+  ];
+
+  meta = {
+    description = "Colcon extension for collecting coverage.py results";
+    homepage = "https://colcon.readthedocs.io/";
```

That page is for colcon in general.
```suggestion
    homepage = "https://github.com/colcon/colcon-coveragepy-result";
```